src/charts/CytoViz/CytoViz.js

```diff
@@ -184,25 +187,38 @@ export const CytoViz = (props) => {
             textColor="primary"
             aria-label="Cytoscape visualization side drawer"
           >
-            <Tab icon={<AccountTreeIcon />} label={labels_.elementDetails} />
-            <Tab icon={<SettingsIcon />} label={labels_.settings.title} />
+            <Tab
+              data-cy="cytoviz-drawer-details-tab-button"
+              icon={<AccountTreeIcon />}
+              label={labels_.elementDetails}
+            />
+            <Tab data-cy="cytoviz-drawer-settings-tab-button" icon={<SettingsIcon />} label={labels_.settings.title} />
           </Tabs>
-          <IconButton onClick={closeDrawer}>
+          <IconButton data-cy="cytoviz-close-drawer-button" onClick={closeDrawer}>
             <ChevronLeftIcon />
           </IconButton>
         </div>
         <div className={classes.drawerContent}>
-          <TabPanel value={currentDrawerTab} index={0}>
+          <TabPanel data-cy="cytoviz-drawer-details-tab-content" value={currentDrawerTab} index={0}>
             {currentElementDetails || labels_.noSelectedElement}
           </TabPanel>
-          <TabPanel value={currentDrawerTab} index={1}>
+          <TabPanel data-cy="cytoviz-drawer-settings-tab-content" value={currentDrawerTab} index={1}>
             <div className={classes.settingsContainer}>
               <div className={classes.settingLine}>
                 <div className={classes.settingLabel}>{labels_.settings.layout}</div>
                 <div className={classes.settingInputContainer}>
-                  <Select value={currentLayout} onChange={changeCurrentLayout} inputProps={{ 'aria-label': 'Layout' }}>
+                  <Select
+                    data-cy="cytoviz-layout-selector"
+                    value={currentLayout}
+                    onChange={changeCurrentLayout}
+                    inputProps={{ 'aria-label': 'Layout' }}
+                  >
                     {DEFAULT_LAYOUTS.concat(Object.keys(extraLayouts)).map((layoutName) => (
-                      <MenuItem key={layoutName} value={layoutName}>
+                      <MenuItem
+                        data-cy={`cytoviz-layout-item-${layoutName.toLowerCase()}`}
```

Comment:
I'm not a great fan of `str-${var}` string concatenation form...

Reply:
The [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates) syntax quite powerful though, we can talk about this in a tech assembly if you want :slightly_smiling_face: